python/cudf/cudf/core/multiindex.py

```diff
@@ -1131,7 +1131,7 @@ def values(self):
             [4, 2],
             [5, 1]])
         >>> type(midx.values)
-        <class 'cupy._core.core.ndarray'>
+        <class 'cupy.ndarray'>
```

Comment:
This _requires_ that we move to cupy 11, whereas all the other changes just enable it.

Reply:
@bdice is there a way to version the doctest or do you think it will be an overkill and should we just skip testing just this docstring?

Reply:
You can use an ellipsis to make it pass for both versions. Doctest will allow any content in the ellipsis.
```suggestion
        <class 'cupy...'>
```

https://github.com/rapidsai/cudf/blob/096bbc45272dc81b31a649b8b82159a483e77495/python/cudf/cudf/tests/test_doctests.py#L83-L87

https://docs.python.org/3/library/doctest.html#doctest.ELLIPSIS

Reply:
Thanks, went for `<class 'cupy...ndarray'>` to ensure we're getting the type of object we expect back.